style.css

```diff
@@ -1,136 +1,94 @@
-ul {
-    list-style: none;
-    margin: 0;
-    padding-left: 0;
-    justify-content: right;
-    padding-right: 8px;
-    padding-right: 50px;
-    
+.logo img {
+    width: 107.54px;
+    height: 16.71px;
 }
 
-a {
-    text-decoration: none;
-    color: #4D4D4D;
-    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
-    vertical-align: middle;
-    font-size:small;
-    line-height: 1%;
-
-
+.header {
+    background-color: white;
+    justify-content: space-between;
+    display: flex;
 }
 
-li {
-    display: inline;
-    padding-bottom: 200px;
-    
+body {
+    background-color: black;
 }
 
+ul li {
+    display: inline-block;
+}
```

Comment:
By this style, you override all ul in the project. If you want styles to be header only, specify them by class. Check out all .css files for problems like this